ppa/archive/hathi.py

```diff
@@ -269,9 +269,32 @@ def content_dir(self):
     def pairtree_client(self):
         """Initialize a pairtree client for the pairtree datastore this
         object belongs to, based on its Hathi prefix id."""
+        store_dir = os.path.join(settings.HATHI_DATA, self.pairtree_prefix)
+        ptree_file_prefix = f"{self.pairtree_prefix}."
```

Comment:
What do you think about moving the logic and correcting the behavior of the `self.pairtree_prefix` property so it includes the '.' as it should? You'll have to update a few unit tests, but it looks like it is only used here. I guess you'd then need to update the logic to remove it when generating the path for the `store_dir`.

Defer to you on which is preferable, I'm fine with either.